src/vector/app.tsx

```diff
@@ -168,16 +168,14 @@ async function verifyServerConfig(): Promise<IConfigOptions> {
         const incompatibleOptions = [wkConfig, serverName, hsUrl].filter((i) => !!i);
         if (incompatibleOptions.length > 1) {
             // noinspection ExceptionCaughtLocallyJS
-            throw newTranslatableError(
-                _td(
-                    "Invalid configuration: can only specify one of default_server_config, default_server_name, " +
-                        "or default_hs_url.",
-                ),
+            throw new UserFriendlyError(
+                "Invalid configuration: can only specify one of default_server_config, default_server_name, " +
+                    "or default_hs_url.",
             );
         }
         if (incompatibleOptions.length < 1) {
             // noinspection ExceptionCaughtLocallyJS
-            throw newTranslatableError(_td("Invalid configuration: no default server specified."));
+            throw new UserFriendlyError("Invalid configuration: no default server specified.");
```

Comment:
this new error looks interesting for us : will it help at translate server errors?